_posts/en/newsletters/2023-01-18-newsletter.md

```diff
@@ -0,0 +1,216 @@
+    taproot], for example, which would conceal the operation of the
+    vault---a very nice feature".
+
+    Separately, Anthony Towns [notes][towns op_vault] that the proposal
+    allows the vault user to freeze their funds at any time just by
+    spending the funds to the address for the highly trusted spending
+    path, allowing the user to unfreeze their funds later.  This
+    benefits the vault owner because they don't need to access any
+    specially-secured key material, such as a cold wallet, in order to
+    block a theft attempt.  However, any third party who learns the
+    address can also freeze the user's funds (although they'll have to
+    pay a transaction fee to do so), creating an inconvenience for the
+    user.  Given that many lightweight wallets disclose their addresses
+    to third parties in order to locate their onchain transactions,
+    vaults built on those wallets might unintentionally give third
+    parties the ability to grief vault users.  Towns suggests an
+    alternative construction for the freezing condition that would
+    require providing an additional piece of non-private information in
+    order to initiate a freezing, preserving the benefits of the scheme
+    but also reducing the risk that a wallet would unnecessarily give
+    third parties the ability to freeze funds and grief the user.  Towns
+    also suggests a possible improvement in batching support and
+    contemplates taproot support.
+
+    Several replies also mentioned that `OP_UNVAULT` can provide many of
+    the features of the [OP_CHECKTEMPLATEVERIFY][topic
+    op_checktemplateverify] (CTV) proposal, including the benefits to
+    [DLCs][topic dlc] previously described in [Newsletter #185][news185
+    ctv-dlc], although at greater onchain cost than directly using CTV.
+
+    As of this writing, discussion on the mailing list was no longer
+    active but we expect the idea will continue to be evaluated
+    alongside other covenant and vault proposals.
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+- **Kraken announces sending to taproot addresses:**
+  In a recent [blog post][kraken bech32m], Kraken announced they support
+  withdrawing (sending) to [bech32m][topic bech32] addresses.
+
+- **Whirlpool coinjoin rust client library announced:**
+  The [Samourai Whirlpool Client][whirlpool rust client] is a rust library for
+  interacting with the Whirlpool [coinjoin][topic coinjoin] platform.
+
+- **Ledger supports miniscript:**
+  Ledger's Bitcoin firmware v2.1.0 release for its hardware signing
+  devices supports [miniscript][topic miniscript] announced [previously][ledger miniscript].
+
+- **Liana wallet released:**
+  The first version of the Liana wallet was [announced][liana blog] that
+  supports singlesig wallets with a [timelocked][topic timelocks] recovery key.
```

Comment:
Something like this? The sentence didn't quite read right to me
```suggestion
  The first version of the Liana wallet was [announced][liana blog]. The wallet
  supports singlesig wallets with a [timelocked][topic timelocks] recovery key.
```

Reply:
Thanks @glozow, took this suggestion